<launch>
	<!-- values passed by command line input -->
	<arg name="model" default="dummy.urdf" />
	<arg name="gui" default="True" />

	<!-- set these parameters on Parameter Server -->
	<param name="robot_description" textfile="$(find dummy_ros)/urdf/$(arg model)" />
	<!--<param name="use_gui" value="$(arg gui)"/>-->

	<node name="invert_kinematics_node" pkg="dummy_ros" type="invert_kinematics_node.py" />
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find dummy_ros)/rviz/dummy.rviz" required="true" />
        <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
	      <param name="port" value="tcp"/>
	</node>
</launch>
